src/internal/tester/aw/assert_wrapper.go

```diff
@@ -0,0 +1,83 @@
+package aw
+
+import (
+	"github.com/alcionai/clues"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// this file is subfoldered to avoid cyclic dependencies
+
+// ---------------------------------------------------------------------------
+// require wrappers
+// ---------------------------------------------------------------------------
+
+// MustErr wraps require.Error()
+func MustErr(t require.TestingT, err error, etc ...any) {
```

Comment:
I feel like we need to be careful doing stuff like this. Testify has a large surface area when it comes to functions and this only covers the `Error` cases, not the `Errorf` cases which expands the surface area even more. Is there some other way we can get the information we need? For example, does testify print the errors with the `%+v` specifier and can we leverage that? Overall, having to wrap test functions to get the info we need feels like a bit of an anti-pattern

Reply:
Yeah, I'm not particularly happy about it, either.  But we don't have much tooling to work with here.  Fwiw, here's testify's Error func:
```go
func Error(t TestingT, err error, msgAndArgs ...interface{}) bool {
	if err == nil {
		if h, ok := t.(tHelper); ok {
			h.Helper()
		}
		return Fail(t, "An error is expected but got nil.", msgAndArgs...)
	}

	return true
}
```
So it _does_ include `%+v`.  But I'm not certain this is the best way to display the data involved.  Without some preprocessing trigger (ex: `assert.Error(t, clues.Dump(err))`) we'd end up interpolating structured data throughout stack trace lines.  Maybe that's not the worst thing, if the caller is already requesting additional formatting.  It does deviate from standard expectations about stack trace design.  Much easier compliance that way, though.  I'll ask around for additional advice and noodle on the solution.

Reply:
Hmm, I think the most idiomatic way to do this is to append the data to the assert call.  Less fancy, but more proper.  going to close this PR and redo in a different branch.